vendor/modules.txt

```diff
@@ -205,6 +205,7 @@ github.com/hashicorp/terraform-plugin-framework/provider/schema
 github.com/hashicorp/terraform-plugin-framework/providerserver
 github.com/hashicorp/terraform-plugin-framework/resource
 github.com/hashicorp/terraform-plugin-framework/resource/schema
+github.com/hashicorp/terraform-plugin-framework/resource/schema/booldefault
```

Comment:
did this end up being used?  I can't find it in the source.

Reply:
This got introduced in first iteration of development. I missed to run `go mod vendor` after clean up. Thanks for pointing it out!